runtime/compiler/z/codegen/S390J9CallSnippet.hpp

```diff
@@ -117,11 +117,16 @@ class S390VirtualUnresolvedSnippet : public TR::S390VirtualSnippet
    virtual uint8_t *emitSnippetBody();
 
    virtual uint32_t getLength(int32_t estimatedSnippetStart);
+   uint8_t* getJ2IThunk() { return thunkAddress; }
```

Comment:
Can we rename this to `getJ2IThunkAddress`? It really is extracting the address and perhaps even make it a `void*`?

Reply:
OK. I'll change this variable and getter to `void*`, together with some relevant `J9S390PrivateLinkage`  code that has to do with thunk address.